pkgs/misc/vim-plugins/update.py

```diff
@@ -413,6 +416,11 @@ def generate_nix(plugins: List[Tuple[str, str, Plugin]], outfile: str):
     print(f"updated {outfile}")
 
 
+def commit_changes(repo: git.Repo, *files: Path):
```

Comment:
I think `files` should be an explicit list instead.

Reply:
I knew you would and changed it in a subsequent commit.